RecFCCeeCalorimeter/tests/options/ALLEGRO_o1_v03.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# set-up the Key4hep environment if not already set
+if [[ -z "${KEY4HEP_STACK}" ]]; then
+  source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh
+else
+  echo "The Key4hep stack was already loaded in this environment."
+fi
+
+# download the events to reconstruct
+if ! -test https://fccsw.web.cern.ch/fccsw/filesForSimDigiReco/gen/pythia_ee_z_qq_10evt.hepmc; then
+  echo "Downloading files needed for simulation"
+  wget https://fccsw.web.cern.ch/fccsw/filesForSimDigiReco/gen/pythia_ee_z_qq_10evt.hepmc
+fi
+# run the SIM step (for debug do not run it if files already present. Comment the if and fi lines for production)
+#if ! test -f ALLEGRO_sim_ee_z_qq.root; then
+echo "Performing the Geant4 simulation with ddsim"
+ddsim --inputFiles pythia_ee_z_qq_10evt.hepmc --numberOfEvents -1 --outputFile ALLEGRO_sim_ee_z_qq.root --compactFile $K4GEO/FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml
```

Comment:
If `ddsim` fails will the script be run until the end? I mean will `k4run` be run on non-valid files?

Reply:
I added in a new commit some extra code to exit if either the download of the input hempc file or the simulation step fail